docs/Modules/multichain.mdx

```diff
@@ -1,22 +1,153 @@
 ---
+sidebar_label: "Multichain Validation Module"
 sidebar_position: 3
+description: "Overview of Multichain Validation Module with ECDSA Ownership"
 ---
 
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
 # Multichain Validation Module
 
-This module acts as an extention of the [ECDSA Module](/Modules/ecdsa). The added benefit of this module is you now have the ability to use a single signer accross multiple blockchain networks. This allows you now to completely abstract away the chain that the user is interacting with.
+This module acts as an extention of the [ECDSA Module](/Modules/ecdsa). The added benefit of this module is you now have the ability to sign once and authorize any amount of userOps accross multiple blockchain networks.
 
-## Installation
+### What is the Multichain Validation Module? 🤔
+
+This module allows **Externally Owned Accounts (EOAs)** to authorize and sign user operations (UserOps) for Biconomy Smart Accounts on Multiple chains with single ECDSA signature. 
+
+`Benefits`
+- significantly improves UX for deploying and setting up Smart Accounts on several chains 
+- significantly reduces user friction for multichain operations.
+
+:::tip
+You can use also this module as the **default module** just like ECDSA Validation Module as it has all it's capabilities for sending ops on particular chain.
+:::
+
+#### Key Functions 🔐
+
+- **ECDSA Signature Scheme**: Utilizes the ECDSA secp256r1 curve for secure signing.
+- **Flexible Signer Authentication**: Supports various signer solutions like Privy, Fireblocks, Arcana Auth, Capsule, Lit protocol, Turnkey, Web3Auth, Particle, Magic, Portal etc
+- **EIP-1271 Compliance**: Allowing Smart Accounts to sign Ethereum messages for logging into dApps.
+- **Sign once, Execute on multiple chains**
+
+#### Use Cases 🌟
+
+Enables use cases which require several actions to be authorized for several chains with just one signature required from user.
+
+1. **Multichain intent for example exit all my native tokens for usdc
+2. **Allows a dApp to require just one signature from their user to configure & deploy smart accounts on multiple chains and delegate certain actions with permission via session keys on all those chains
+3. **Cross chain state sync actions e.g transfer ownership of SA, change ens, similar actions on protocol that is deployed on multiple chains
+4. **Onboard user on x chains by deploying their smart account + some action like register in raffle 
+
+### How does it work? 🤔
+
+1. We take all the userOps which we need to sign.
```

Comment:
can we rephrase these steps
It involves collecting the set of userOps for signing, 
each userOp Hash is a leaf in a Merkle Tree.
Instead of signing individual hashes, user signs the Merkle Root.
This signed Merkle Root serves as on-chain evidence, proving any userOp within the tree is authorized.

Reply:
Agree.

Reply:
check the changes

Reply:
### How does it work? 🤔

1. Collecting the set of userOps for signing
2. Calculate userOpHash
3. Every userOpHash is now a leaf of a Merkle Tree.
4. Instead of signing individual hashes, user signs the Merkle Root.

Now this signed Merkle Root serves as on-chain evidence, proving any userOp within the tree is authorized. User can authorize any amount of userOps with just one signature over the Merkle Root of those userOps